src/common/constants.ts

```diff
@@ -64,3 +64,9 @@ export const defaultNetworks: Networks = {
     chainId: ChainID.Testnet,
   },
 } as const;
+
+export enum QueryRefreshRates {
+  SLOW = 30_000,
+  QUICK = 10_000,
+  FAST = 3_500,
+}
```

Comment:
Does _fast_ really imply one up than _quick_?

Reply:
lol i was trying to avoid FAST and VERY_FAST

I'd love for them to be semantic, implying some use cases, eg

NOT_IMPORTANT
MODERATELY_IMPORTANT
IMPORTANT

basically I want the "FAST" version to be "REAL_TIME" and others to be slower named items. what would you suggest?

Reply:
Medium would probably do for the time being 😅